config/linux-settings.gypi

```diff
@@ -71,7 +71,7 @@
 					'-Wno-unused-parameter',	# Just contributes build noise
 					'-Werror=return-type',
 					'-Werror=uninitialized',
-					'-Wno-error=maybe-uninitialized',
+					#'-Wno-error=maybe-uninitialized',
```

Comment:
It's GCC only but linux-settings is used on android and emscripten so may well fail on those